src/cascade/model/model.py

```diff
@@ -122,20 +164,31 @@ def from_var(cls, var, parent_location, weights=None, covariates=None):
                 in the keys.
             weights (Dict[Var]): Population weights for integrands.
             covariates(List[Covariate]): Covariate objects.
+            multiple_random_effects (bool): Create a separate smooth grid for
+                each random effect in the Var. The default is to create a
+                single smooth grid for all random effects.
 
         Returns:
             Model: with Uniform distributions everywhere and no mulstd.
         """
         child_locations = [k[1] for k in var.random_effect.keys()]
         nonzero_rates = list(var.rate.keys())
-        model = cls(nonzero_rates, parent_location, child_locations, weights, covariates)
+        model = cls(nonzero_rates, parent_location, child_locations, weights=weights, covariates=covariates)
```

Comment:
The arguments were in the wrong order. This was a bug in the previous commit. Using keywords to avoid that.